lib/studies/reply/adaptive_nav.dart

```diff
@@ -1215,3 +1222,33 @@ class _FadeThroughTransitionSwitcher extends StatelessWidget {
     );
   }
 }
+
+class _SharedAxisTransitionSwitcher extends StatelessWidget {
+  const _SharedAxisTransitionSwitcher({
+    @required this.defaultChild,
+  }) : assert(defaultChild != null);
+
+  final Widget defaultChild;
+
+  @override
+  Widget build(BuildContext context) {
+    return Selector<EmailStore, bool>(
+      builder: (context, onSearchPage, child) {
```

Comment:
I think the `builder` arg should go after the `selector` one (similar to putting `child` last)

Reply:
Thanks will do!